### PR TITLE
Fix IRT key in annotations

### PR DESCRIPTION
**Pull request by MaximPlusov**

### CODE REVIEW

**Comment by petervwyatt** [comment]
@MaximPlusov 
The Adobe PDF Reference for PDF 1.5 describes **IRT** in section 8.4 Annotation States. However the key **IRT** was only explicitly listed in Table 8.19 for Text Annots. The index in PDF 1.5 also explicitly stated "**IRT** entry (text annotation dictionary)" and thus limited it to text annots. The PDF 1.5 errata (8/28/2003) doesn't mention **IRT** either so no help there. The Adobe PDF Reference for PDF 1.6 then has IRT listed in table 8.17 Markup annots with version 1.5 implying that it was actually for all markup annots and not just text annots. 
So I think the correct thing for Arlington is to have AnnotText have IRT with PDF 1.5 while all the other markup annots get **IRT** introduced with PDF 1.6 (when **RT** was also introduced).
Would you agree?

**Comment by MaximPlusov** [comment]
Yes, I agree. That's what I did in this PR

File: tsv/latest/AnnotText.tsv [hunk 1]
```diff
@@ -22,7 +22,7 @@ T	string-text	1.1		FALSE	FALSE	FALSE
...
 RC	stream;string-text	1.5		FALSE	[TRUE];[FALSE]	FALSE				[Stream];[]	
 CreationDate	date	1.5		FALSE	FALSE	FALSE					
-IRT	dictionary	1.5		fn:IsRequired(fn:IsPresent(RT))	FALSE	FALSE				[AnnotText,AnnotLink,AnnotFreeText,AnnotLine,AnnotSquare,AnnotCircle,AnnotPolygon,AnnotHighlight,AnnotUnderline,AnnotSquiggly,AnnotStrikeOut,AnnotCaret,AnnotStamp,AnnotInk,AnnotPopup,AnnotFileAttachment,AnnotSound,AnnotMovie,AnnotScreen,AnnotWidget,AnnotPrinterMark,fn:SinceVersion(1.6,AnnotWatermark),fn:SinceVersion(1.6,Annot3D),fn:SinceVersion(1.7,AnnotRedact),fn:SinceVersion(2.0,AnnotProjection),fn:SinceVersion(2.0,AnnotRichMedia)]	
+IRT	dictionary	1.5		fn:IsRequired(1.5,fn:IsPDFVersion(fn:IsPresent(State)) || fn:SinceVersion(1.6,fn:IsPresent(RT)))	FALSE	FALSE				[AnnotText,AnnotLink,AnnotFreeText,AnnotLine,AnnotSquare,AnnotCircle,AnnotPolygon,AnnotHighlight,AnnotUnderline,AnnotSquiggly,AnnotStrikeOut,AnnotCaret,AnnotStamp,AnnotInk,AnnotPopup,AnnotFileAttachment,AnnotSound,AnnotMovie,AnnotScreen,AnnotWidget,AnnotPrinterMark,fn:SinceVersion(1.6,AnnotWatermark),fn:SinceVersion(1.6,Annot3D),fn:SinceVersion(1.7,AnnotRedact),fn:SinceVersion(2.0,AnnotProjection),fn:SinceVersion(2.0,AnnotRichMedia)]
```
Comment (petervwyatt) [inline tsv/latest/AnnotText.tsv:25]: `fn:IsRequired(1.5,fn:IsPDFVersion(fn:IsPresent(State)) || fn:SinceVersion(1.6,fn:IsPresent(RT)))`
should be
`fn:IsRequired(fn:IsPDFVersion(1.5,fn:IsPresent(State)) || fn:SinceVersion(1.6,fn:IsPresent(RT)))`

**Review by petervwyatt — CHANGES_REQUESTED** [review]

File: tsv/latest/AnnotText.tsv (hunk 1 above)
Comment (MaximPlusov) [reply tsv/latest/AnnotText.tsv]: Done

**Pull request merged**